include/rocksdb/file_system.h

```diff
@@ -112,14 +112,19 @@ struct IOOptions {
   // fsync, set this to force the fsync
   bool force_dir_fsync;
 
+  // Can be used by underlying file systems to skip sub directories and list
+  // only files in GetChildren API.
+  bool list_files_only;
```

Comment:
By default, does GetChildren do a recursive listing, or does it just list sub-directories in the top level directory but not recurse into them? If its the former, maybe this option should be called `do_not_recurse` as `list_files_only` doesn't make it clear whether to recurse or not.

Reply:
Makes sense. I think, do_not_recurse does seem to be a cleaner name.